libs/application/templates/financial-aid/src/fields/StudentForm/studentForm.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react'
+import { Box, Text } from '@island.is/island-ui/core'
+import { FAFieldBaseProps, TwoTypeAnswers } from '../../lib/types'
+import { useIntl } from 'react-intl'
+import { studentForm } from '../../lib/messages'
+import { InputController, RadioController } from '@island.is/shared/form-fields'
+import { useFormContext } from 'react-hook-form'
+
+const StudentForm = ({ errors, application }: FAFieldBaseProps) => {
+  const { formatMessage } = useIntl()
+  const { answers } = application
+
+  const typeInput = {
+    id: 'student.isStudent',
+    error: errors?.student?.isStudent,
+  }
+  const customeInput = {
+    id: 'student.custom',
+    error: errors?.student?.custom,
+  }
+  const { clearErrors, getValues } = useFormContext()
+
+  return (
+    <>
+      <Box marginTop={[2, 2, 3]}>
+        <RadioController
+          id={typeInput.id}
+          defaultValue={answers?.student?.isStudent}
+          options={[
+            {
+              value: TwoTypeAnswers.No,
+              label: formatMessage(studentForm.form.notStudent),
+            },
+            {
+              value: TwoTypeAnswers.Yes,
+              label: formatMessage(studentForm.form.isStudent),
+            },
+          ]}
+          largeButtons
+          backgroundColor="white"
+          error={typeInput.error}
+        />
+      </Box>
+      {getValues(typeInput.id) === TwoTypeAnswers.Yes && (
+        <Box>
+          <InputController
+            id={customeInput.id}
```

Comment:
waiting for homecircumstance PR to do this via classnames

Reply:
it is approved